### DISCUSSION TITLE
Best practice to add fonts into a web component

### DISCUSSION THREAD

**Discussion by 2conmeo** [rusianh/eShopProject_MVC_API#4] [Q&A]
I'm using: `npm init @open-wc` to create a brand new web component (not an application) called my-foo this component will use [mwc-icon-button](https://github.com/material-components/material-components-web-components/tree/master/packages/icon-button). The instructions say that I need to import the Material Icon font. What's the correct way of doing so using the generated code?

**Comment by rusianh** [comment]
```
<link href="https://fonts.googleapis.com/css?family=Material+Icons&display=block" rel="stylesheet">

<mwc-icon>shopping_cart</mwc-icon>

<script type="module">
  import '@material/mwc-icon';
</script>
```

**Comment by rusianh** [comment]
class Modal extends Lit {
private _eventEmitter = emitPreventableEvent(this);

private emitCloseModal(){
   this._eventEmitter('modal-close', {detail: true}, () => {
      this.open = false;
   });
}

render() {
   return html`
   ...stuff

   <button class="close-modal" @click="emitCloseModal">X</button>

   ...stuff
   `;
}
